docs/DatabaseCatalogAPI.yaml

```diff
@@ -367,8 +385,6 @@ components:
           properties:
             locations:
               $ref: "#/components/schemas/Locations"
-        - type: object
-          properties:
             latest_dataset:
```

Comment:
Unrelated to this PR. I fixed this "extra" type declaration as a consumer of the file might have issues rendering the yaml file or generating code.